### PR TITLE
Introducing the Activity Bar, initial concept and layout

### PR DESCRIPTION
**Pull request by guerler**
This PR introduces the new Activity Bar with a focus on the initial concept and structure. The functionality is limited but provides a platform for further extension which we intend to split between different developers. In follow-ups we intend e.g. to allow user customization i.e. choose which activities to show, add a client-store and a new API endpoint, display notifications and refine the relationship between the masthead and the activity bar to avoid duplication and more.

This PR also required and includes a new SidePanel and a re-usable Popper components. Until now SidePanels relied on explicit styles to enforce size control, this however does not play well with the `flex` layout features. The new SidePanels do look and feel similar but do not rely on extensive style declarations anymore. They are currently only applied to the Analysis panel and the Published pages. In a follow-up these flex-based side panels should be applied to the Workflow Editor, and the Admin page. This should allow us to strip the legacy panel helpers from the code base.

NOTE: The Activity Bar is hidden behind a switch with is made available at the bottom of the User Tab in the Masthead.

https://user-images.githubusercontent.com/2105447/231701827-07cc7357-509c-4608-89c1-0a5a86d95bcb.mov

While in beta, the new activity bar is hidden behind a switch.

## How to test 

the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

**Comment by martenson** [comment]
has potential imho

**Comment by guerler** [comment]
Another variation:

image

**Comment by guerler** [comment]
Additional view of the upload progress bar:
image

**Comment by guerler** [comment]
Restoring labels in center tabs:

image

**Comment by guerler** [comment]
Summarizing today's team meeting, the consensus is to explore additional options which leave the left side panel intact (i.e. don't move it to a popup) but instead allow to collapse it into a vertical secondary masthead / icon bar, containing a search icon tab at the top. This would avoid having to place everything into a single masthead for now. Additional suggestions are to 1) evaluate how to deploy it i.e. A/B testing and/or a beta-config switch, 2) plan to allow customization through user preferences, 3) and consider generalizations using window managers.

**Comment by guerler** [comment]
Alternative draft with additional side panel just to keep track:
image

**Comment by guerler** [comment]
Other variations explored during this exercise:
Display entire masthead as vertical icon bar:
image
Adding tool panel as popover to masthead:
image

File: config/plugins/tours/core.galaxy_ui.yaml [hunk 1]
```diff
@@ -13,8 +13,8 @@ steps:
       # 'content' is the actual text that is shown to the user
       content: "This short tour will guide you through Galaxy's user interface. You can leave the tour at any time by pressing escape."
 
-    # 'element' is the JQuery Selector (http://api.jquery.com/category/selectors/) of the element you want to describe
-    # In this case we want to highlight the Upload button with the `.upload-button` selector
+    # 'element' is the JQuery Selector (http://api.jquery.com/category/selectors/) of the element you want to describe.
+    # We could highlight the Upload button using `#activity-upload > a` or, alternatively specify a `component` using the navigation interface.
```
Comment (bgruening) [inline config/plugins/tours/core.galaxy_ui.yaml]: Is it really `> a` or without?

Comment (guerler) [reply config/plugins/tours/core.galaxy_ui.yaml]: Thank you.

**Review by bgruening — APPROVED** [review]
Its a great first pass and works well for me. Thanks Aysam!

**Comment by guerler** [comment]
Thank you!

**Pull request merged**